### PR TITLE
Update cleanup job to use COMIN/COMOUT

### PR DESCRIPTION
**Pull request by HenryRWinterbottom**
# Description

NCO has requested that each COM variable specify whether it is an input or an output. This completes that process for the global-workflow clean-up task.

  Refs #2451 

# Type of change
- Maintenance (code refactor, clean-up, new CI test, etc.)

# Change characteristics
- Is this a breaking change (a change in existing functionality)? NO
- Does this change require a documentation update? NO

# How has this been tested?

- `C48_S2SW` experiment on RDHPCS Hera, modified to include 4 cycles.

# Change characteristics
- Is this a breaking change (a change in existing functionality)? NO
- Does this change require a documentation update? NO

# How has this been tested?

```
       CYCLE                    TASK                       JOBID               STATE         EXIT STATUS     TRIES      DURATION
================================================================================================================================
202103231200             gfsstage_ic                    61083306           SUCCEEDED                   0         1          22.0
202103231200             gfswaveinit                    61083307           SUCCEEDED                   0         1          31.0
202103231200                 gfsfcst                    61083520           SUCCEEDED                   0         1        2299.0
202103231200    gfsatmos_prod_f000-f006                    61083820           SUCCEEDED                   0         1         426.0
202103231200    gfsatmos_prod_f009-f015                    61083821           SUCCEEDED                   0         1         434.0
202103231200    gfsatmos_prod_f018-f024                    61083878           SUCCEEDED                   0         1         381.0
202103231200    gfsatmos_prod_f027-f033                    61083879           SUCCEEDED                   0         1         384.0
202103231200    gfsatmos_prod_f036-f042                    61084008           SUCCEEDED                   0         1         390.0
202103231200    gfsatmos_prod_f045-f051                    61084009           SUCCEEDED                   0         1         395.0
202103231200    gfsatmos_prod_f054-f060                    61084148           SUCCEEDED                   0         1         550.0
202103231200    gfsatmos_prod_f063-f069                    61084332           SUCCEEDED                   0         1         493.0
202103231200    gfsatmos_prod_f072-f078                    61084333           SUCCEEDED                   0         1         477.0
202103231200    gfsatmos_prod_f081-f087                    61084389           SUCCEEDED                   0         1         397.0
202103231200    gfsatmos_prod_f090-f096                    61084390           SUCCEEDED                   0         1         394.0
202103231200    gfsatmos_prod_f099-f105                    61084571           SUCCEEDED                   0         1         392.0
202103231200    gfsatmos_prod_f108-f114                    61084559           SUCCEEDED                   0         1         388.0
202103231200    gfsatmos_prod_f117-f120                    61084718           SUCCEEDED                   0         1         258.0
202103231200              gfstracker                    61084973           SUCCEEDED                   0         1          81.0
202103231200              gfsgenesis                    61084974           SUCCEEDED                   0         1         541.0
202103231200    gfsocean_prod_f006-f018                    61083924           SUCCEEDED                   0         1          79.0
202103231200    gfsocean_prod_f024-f036                    61084049           SUCCEEDED                   0         1          83.0
202103231200    gfsocean_prod_f042-f054                    61084181           SUCCEEDED                   0         1          93.0
202103231200    gfsocean_prod_f060-f072                    61084336           SUCCEEDED                   0         1         155.0
202103231200    gfsocean_prod_f078-f090                    61084471           SUCCEEDED                   0         1          84.0
202103231200    gfsocean_prod_f096-f108                    61084597           SUCCEEDED                   0         1          89.0
202103231200    gfsocean_prod_f114-f120                    61084598           SUCCEEDED                   0         1          55.0
202103231200    gfsice_prod_f006-f018                    61083925           SUCCEEDED                   0         1          59.0
202103231200    gfsice_prod_f024-f036                    61084050           SUCCEEDED                   0         1          63.0
202103231200    gfsice_prod_f042-f054                    61084182           SUCCEEDED                   0         1          62.0
202103231200    gfsice_prod_f060-f072                    61084337           SUCCEEDED                   0         1          86.0
202103231200    gfsice_prod_f078-f090                    61084472           SUCCEEDED                   0         1          69.0
202103231200    gfsice_prod_f096-f108                    61084599           SUCCEEDED                   0         1          63.0
202103231200    gfsice_prod_f114-f120                    61084600           SUCCEEDED                   0         1          44.0
202103231200          gfswavepostsbs                    61083681           SUCCEEDED                   0         1        2427.0
202103231200          gfswavepostpnt                    61084601           SUCCEEDED                   0         1       11260.0
202103231200                 gfsarch                    61092509           SUCCEEDED                   0         1          48.0
202103231200              gfscleanup                    61188921           SUCCEEDED                   0         1          26.0
================================================================================================================================
202103241200             gfsstage_ic                          -1           SUCCEEDED                   0         1           0.0
202103241200             gfswaveinit                          -1           SUCCEEDED                   0         1           0.0
202103241200                 gfsfcst                          -1           SUCCEEDED                   0         1           0.0
202103241200    gfsatmos_prod_f000-f006                          -1           SUCCEEDED                   0         1           0.0
202103241200    gfsatmos_prod_f009-f015                          -1           SUCCEEDED                   0         1           0.0
202103241200    gfsatmos_prod_f018-f024                          -1           SUCCEEDED                   0         1           0.0
202103241200    gfsatmos_prod_f027-f033                          -1           SUCCEEDED                   0         1           0.0
202103241200    gfsatmos_prod_f036-f042                          -1           SUCCEEDED                   0         1           0.0
202103241200    gfsatmos_prod_f045-f051                          -1           SUCCEEDED                   0         1           0.0
202103241200    gfsatmos_prod_f054-f060                          -1           SUCCEEDED                   0         1           0.0
202103241200    gfsatmos_prod_f063-f069                          -1           SUCCEEDED                   0         1           0.0
202103241200    gfsatmos_prod_f072-f078                          -1           SUCCEEDED                   0         1           0.0
202103241200    gfsatmos_prod_f081-f087                          -1           SUCCEEDED                   0         1           0.0
202103241200    gfsatmos_prod_f090-f096                          -1           SUCCEEDED                   0         1           0.0
202103241200    gfsatmos_prod_f099-f105                          -1           SUCCEEDED                   0         1           0.0
202103241200    gfsatmos_prod_f108-f114                          -1           SUCCEEDED                   0         1           0.0
202103241200    gfsatmos_prod_f117-f120                          -1           SUCCEEDED                   0         1           0.0
202103241200              gfstracker                          -1           SUCCEEDED                   0         1           0.0
202103241200              gfsgenesis                          -1           SUCCEEDED                   0         1           0.0
202103241200    gfsocean_prod_f006-f018                          -1           SUCCEEDED                   0         1           0.0
202103241200    gfsocean_prod_f024-f036                          -1           SUCCEEDED                   0         1           0.0
202103241200    gfsocean_prod_f042-f054                          -1           SUCCEEDED                   0         1           0.0
202103241200    gfsocean_prod_f060-f072                          -1           SUCCEEDED                   0         1           0.0
202103241200    gfsocean_prod_f078-f090                          -1           SUCCEEDED                   0         1           0.0
202103241200    gfsocean_prod_f096-f108                          -1           SUCCEEDED                   0         1           0.0
202103241200    gfsocean_prod_f114-f120                          -1           SUCCEEDED                   0         1           0.0
202103241200    gfsice_prod_f006-f018                          -1           SUCCEEDED                   0         1           0.0
202103241200    gfsice_prod_f024-f036                          -1           SUCCEEDED                   0         1           0.0
202103241200    gfsice_prod_f042-f054                          -1           SUCCEEDED                   0         1           0.0
202103241200    gfsice_prod_f060-f072                          -1           SUCCEEDED                   0         1           0.0
202103241200    gfsice_prod_f078-f090                          -1           SUCCEEDED                   0         1           0.0
202103241200    gfsice_prod_f096-f108                          -1           SUCCEEDED                   0         1           0.0
202103241200    gfsice_prod_f114-f120                          -1           SUCCEEDED                   0         1           0.0
202103241200          gfswavepostsbs                          -1           SUCCEEDED                   0         1           0.0
202103241200          gfswavepostpnt                          -1           SUCCEEDED                   0         1           0.0
202103241200                 gfsarch                          -1           SUCCEEDED                   0         1           0.0
202103241200              gfscleanup                    61188920           SUCCEEDED                   0         1          27.0
```

# Checklist
- [x] Any dependent changes have been merged and published
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings
- [x] New and existing tests pass with my changes
- [x] I have made corresponding changes to the documentation if necessary

### CODE REVIEW

File: scripts/exglobal_cleanup.sh [hunk 1]
```diff
@@ -59,10 +59,11 @@ for (( current_date=first_date; current_date <= last_date; \
         # TODO: This needs to be revamped to not look at the rocoto log.
         # shellcheck disable=SC2312
         if [[ $(tail -n 1 "${rocotolog}") =~ "This cycle is complete: Success" ]]; then
-            YMD="${current_PDY}" HH="${current_cyc}" declare_from_tmpl COM_TOP
-            if [[ -d "${COM_TOP}" ]]; then
+            YMD="${current_PDY}" HH="${current_cyc}" declare_from_tmpl \
+		    COMOUT_TOP:COM_TOP_TMPL
```
Comment (WalterKolczynski-NOAA) [inline scripts/exglobal_cleanup.sh]: Line continuations should be indented
```suggestion
		        COMOUT_TOP:COM_TOP_TMPL
```

**Review by WalterKolczynski-NOAA — CHANGES_REQUESTED** [review]
One indentation thing.

**Review by WalterKolczynski-NOAA — APPROVED** [review]

**Review by KateFriedman-NOAA — APPROVED** [review]
Looks good!

**Comment by WalterKolczynski-NOAA** [comment]
stmp filled on WCOSS again.

**Comment by WalterKolczynski-NOAA** [comment]
Failure is unrelated to the PR. We need Terry to clean out the restart directory when he gets back Monday.

**Pull request merged**